CSS-Grid-Opdracht/styles.css

```diff
@@ -1,3 +1,60 @@
-#test {
-    color: blue;
+body {
+    background-color: #ffefdb;
+}
+.class-items {
+    background-color: #ba3b44;
+    border: solid;
+    border-width: 2px;
+    border-color: #932f37;
+    padding: 16px;
+    color: white;
+}
+
+.container {
+    display: grid;
+    grid-gap: 16px 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+            "iheader"
+            "ib-aside"
+            "ic-aside"
+            "id-aside"
+            "ie-aside"
+            "."
+            "ifooter";
+}
+
+#xheader {
+    grid-area: iheader;
+}
+#xb-aside {
+    grid-area: ib-aside;
+}
+#xc-aside {
+    grid-area: ic-aside;
+}
+#xe-aside {
+    grid-area: ie-aside;
+}
+#xd-aside {
+    grid-area: id-aside;
+}
+#xfooter {
+    grid-area: ifooter;
+}
+
+@media (min-width: 700px) {
```

Comment:
Ik denk dat je beter vanaf 786px kan werken, want dat is de standaard maat voor tablet en kleine PC's, Zie hier de code gestolen van w3schools: 
` /* Extra small devices (phones, 600px and down) */
@media only screen and (max-width: 600px) {...}

/* Small devices (portrait tablets and large phones, 600px and up) */
@media only screen and (min-width: 600px) {...}

/* Medium devices (landscape tablets, 768px and up) */
@media only screen and (min-width: 768px) {...}

/* Large devices (laptops/desktops, 992px and up) */
@media only screen and (min-width: 992px) {...}

/* Extra large devices (large laptops and desktops, 1200px and up) */
@media only screen and (min-width: 1200px) {...} `